src/schema/contentful/phoenix.js

```diff
@@ -76,6 +76,24 @@ const typeDefs = gql`
     ${entryFields}
   }
 
+  type PersonBlock implements Block {
+    "Name of the person displayed on the block"
+    name: String
+    "The type of employer this person is"
```

Comment:
How's about this?
```suggestion
    "The person's position within the organization"
```

Since this includes members and board members who are not employees.